tensorboard/backend/event_processing/data_ingester.py

```diff
@@ -30,6 +30,7 @@
 from tensorboard.plugins.pr_curve import metadata as pr_curve_metadata
 from tensorboard.plugins.scalar import metadata as scalar_metadata
 from tensorboard.util import tb_logging
+from tensorboard.compat import tf
```

Comment:
We should add a `"//tensorboard/compat:tensorflow"` dep on the corresponding BUILD target for this.

Same goes for the test file.